### PR TITLE
created video- detail page

### PR DESCRIPTION
**Pull request by Rockyraven**
features: 
suggestion card added
video detail page added 
view, video title, description update 
image

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

File: src/context/videoContext.js [hunk 1]
```diff
@@ -4,32 +4,29 @@ import axios from "axios";
...
+      try {
+        const response = await axios.get("/api/videos");
+        setVideoList(response.data.videos);
+      } catch (error) {
+        console.log(error);
+      }
+    };
+    getVideoData();
+  }, []);
+  // console.log(videoList)
```
Comment (surbhi84) [inline src/context/videoContext.js]: you can remove if not needed anymore.

File: src/pages/homepage/HomePage.js [hunk 2]
```diff
@@ -1,31 +1,38 @@
-import { useVideo } from 'context/videoContext'
-import React from 'react'
-import { Sidebar, VideoCard } from '../../component'
-import './homepage.css'
-import Chip from '@mui/material/Chip';
-
+import { useVideo } from "context/videoContext";
+import React from "react";
+import { Sidebar, VideoCard } from "../../component";
```
Comment (surbhi84) [inline src/pages/homepage/HomePage.js:3]: you can write component directly like at other places.

**Review by surbhi84 — APPROVED** [review]
- Liked that you used absolute imports.
- Good job with material ui.
- Readable code.
- Looks mergeable to me.

File: src/context/videoContext.js (hunk 1 above)
Comment (Rockyraven) [reply src/context/videoContext.js]: Ok I will remove it

**Comment by Rockyraven** [comment]
thanku for review

**Pull request merged**